src/ripple/app/hook/impl/applyHook.cpp

```diff
@@ -70,6 +70,17 @@ getTransactionalStakeHolders(STTx const& tx, ReadView const& rv)
 
     switch (tt)
     {
+
+        case ttREMIT: {
+            if (destAcc)
+                ADD_TSH(*destAcc, tshSTRONG);
+
+            if (tx.isFieldPresent(sfInform))
+                ADD_TSH(tx.getAccountID(sfInform), tshWEAK);
+
+            break;
```

Comment:
If URIToken is included, shouldn't the issuer be set to TSH?

Reply:
Issuer is the OTXN Account and so by default, the strong TSH.

Reply:
Yes it should. Thank you